### PR TITLE
feat: Update readme to describe new syntax for metadata

### PR DESCRIPTION
**Pull request by guyrenny**
- Add a new variable to allow specific source code version `source_code_version`
- Update readme to describe new syntax for metadata

# Description

 Fixes #

# How Has This Been Tested?

# Checklist:
- [ ] I have updated the relevant example in the examples directory for the module.
- [ ] I have updated the relevant test file for the module.
- [ ] This change does not affect module (e.g. it's readme file change)

### CODE REVIEW

**Review by royfur — APPROVED** [review]
LGTM

**Review by ryantanjunming — COMMENTED** [review]
some reviews

File: modules/coralogix-aws-shipper/main.tf [hunk 1]
```diff
@@ -207,11 +203,11 @@ module "lambda" {
     DLQ_S3_BUCKET      = var.enable_dlq ? var.dlq_s3_bucket : null
     DLQ_URL            = var.enable_dlq ? aws_sqs_queue.DLQ[0].url : null
     ASSUME_ROLE_ARN    = var.lambda_assume_role_arn
-  }
+  } 
   s3_existing_package = {
     bucket = var.custom_s3_bucket == "" ? "coralogix-serverless-repo-${data.aws_region.this.name}" : var.custom_s3_bucket
-    key    = var.cpu_arch == "arm64" ? "coralogix-aws-shipper.zip" : "coralogix-aws-shipper-x86-64.zip"
-  }
+    key = var.cpu_arch == "arm64" ?  "coralogix-aws-shipper${var.source_code_version != "" ? "-${var.cpu_arch}-${var.source_code_version}" : ""}.zip" : "coralogix-aws-shipper-x86-64${var.source_code_version != "" ? "-${var.cpu_arch}-${var.source_code_version}" : ""}.zip"
```
[the remark this answers is not in the document]
Comment (guyrenny) [reply modules/coralogix-aws-shipper/main.tf:209]: I checked it and didnt see an issue

File: modules/coralogix-aws-shipper/local.tf
[the remark this answers is not in the document]
Comment (guyrenny) [reply modules/coralogix-aws-shipper/local.tf:1]: yes, it worked for me

**Pull request merged**